tests/contrib/hooks/test_gcp_api_base_hook.py

```diff
@@ -45,6 +45,57 @@
     default_creds_available = False
 
 
+class NoForbiddenAfterCount(object):
+    """Holds counter state for invoking a method several times in a row."""
+
+    def __init__(self, count, **kwargs):
+        self.counter = 0
+        self.count = count
+        self.kwargs = kwargs
+
+    def __call__(self):
+        """Raise an Forbidden until after count threshold has been crossed.
+        Then return True.
+        """
+        if self.counter < self.count:
+            self.counter += 1
+            raise Forbidden(**self.kwargs)
+        return True
+
+
+@hook.GoogleCloudBaseHook.quota_retry(wait=tenacity.wait_none())
+def _retryable_test_with_temporare_quota_retry(thing):
+    return thing()
+
+
+class retry_if_temporary_quotaTestCase(unittest.TestCase):
```

Comment:
After [#5522](https://github.com/apache/airflow/pull/5522) is merged, this will raise pylint `invalid-name` error. So I would suggest `TestRetryIfTemporaryQuota`

Reply:
Yes. The name should be different. The first implementation was created when the pylint did not exist in the project. I have not yet updated all changes.

Reply:
I updated this PR.